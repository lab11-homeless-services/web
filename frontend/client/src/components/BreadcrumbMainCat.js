import React from 'react'
import image6 from '../img/education.png'
import image1 from '../img/shelter.png'
import image3 from '../img/food.png'
import image2 from '../img/health-care.png'
import image7 from '../img/legal.png'
import image5 from '../img/outreach.png'
import image4 from '../img/hygiene.png'
import image8 from '../img/employment.png'

const BreadcrumbMainCat = props => {
    if (props.cat === 'shelters') {
        return (
            <div>
                <img src={image1} alt=""/>
            </div>
        )
    }
    if (props.cat === "health_care") {
        return (
            <div>
                <img src={image2} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "food") {
        return (
            <div>
                <img src={image3} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "hygiene") {
        return (
            <div>
                <img src={image4} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "outreach_services") {
        return (
            <div>
                <img src={image5} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "education") {
        return (
            <div>
                <img src={image6} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "legal_administrative") {
        return (
            <div>
                <img src={image7} alt=""/>
            </div>
        ) 
    }
    if (props.cat === "jobs") {
        return (
            <div>
                <img src={image8} alt=""/>
            </div>
        ) 
    }
}

export default BreadcrumbMainCat