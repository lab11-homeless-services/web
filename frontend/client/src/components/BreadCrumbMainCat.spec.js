import React from 'react'
import BreadcrumbMainCat from './BreadcrumbMainCat'
import renderer from 'react-test-renderer'
import { render, fireEvent } from 'react-testing-library'


const testProps = {
    cat: 'shelters'
}

describe('<BreadCrumbMainCat />', () => {
    it('BreadCrumbMainCat renders without crashing', () => {
        render(<BreadcrumbMainCat {...testProps}/>)
    })
    it('matches snapshot' ,() => {
        const tree = renderer.create(<BreadcrumbMainCat {...testProps}></BreadcrumbMainCat>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
    it('renders text SHELTER', () => {
        const {getByText} =  render(<BreadcrumbMainCat {...testProps}/>)
        const shelters = getByText('SHELTER')
        expect(shelters).toBeDefined()
    })
})