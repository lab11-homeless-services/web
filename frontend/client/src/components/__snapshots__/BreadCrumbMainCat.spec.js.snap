// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BreadCrumbMainCat /> matches snapshot 1`] = `
<div>
  <img
    alt=""
    className="sc-bdVaJa kKjsMb"
    src="shelter.png"
  />
  <p
    className="sc-bwzfXH kmzsan"
  >
    SHELTER
  </p>
</div>
`;
